api/src/reconciliation/reconciliation.controller.ts

```diff
@@ -487,4 +487,16 @@ export class ReconciliationController {
 
     return logs;
   }
+
+  @Get('/token/:token/environment/:id')
```

Comment:
`token` and `id` are ambiguous here.

This should  be using query params for `GET` calls instead of path parms.

Reply:
Using QueryParams now